config/teleop.yaml

```diff
@@ -70,3 +70,20 @@ teleop:
     blah: ["/tf", "/tf_static", "/rosout"]
     foo: ["/rosout", "/tf"]
     bar: ["/tf_static", "/rosout", "/tf"]
+
+  camera_list: [
+    "Cam1",
+    "Cam2",
+    "Cam3",
+    "Cam4",
+    "Cam5",
+    "Cam6"
```

Comment:
@sashreek1 can you push a commit or tell TJ all the camera names you and Lena and Dan have been using, and then some preset combinations of cameras